tests/providers/amazon/aws/sensors/test_emr_steps.py

```diff
@@ -0,0 +1,190 @@
+        "Jar": "command-runner.jar",
+        "Properties": {},
+    },
+    "Id": "s-VK57YR1Z9Z5N",
+    "Name": "calculate_pi",
+    "Status": {
+        "State": "RUNNING",
+        "StateChangeReason": {},
+        "Timeline": {
+            "CreationDateTime": datetime(2016, 6, 20, 19, 0, 18, tzinfo=tzlocal()),
+            "StartDateTime": datetime(2016, 6, 20, 19, 2, 34, tzinfo=tzlocal()),
+        },
+    },
+}
+
+DESCRIBE_JOB_STEP_CANCELLED_RETURN = {
+    "ActionOnFailure": "CONTINUE",
+    "Config": {
+        "Args": ["/usr/lib/spark/bin/run-example", "SparkPi", "10"],
+        "Jar": "command-runner.jar",
+        "Properties": {},
+    },
+    "Id": "s-VK57YR1Z9Z5N",
+    "Name": "calculate_pi",
+    "Status": {
+        "State": "CANCELLED",
+        "StateChangeReason": {},
+        "Timeline": {
+            "CreationDateTime": datetime(2016, 6, 20, 19, 0, 18, tzinfo=tzlocal()),
+            "StartDateTime": datetime(2016, 6, 20, 19, 2, 34, tzinfo=tzlocal()),
+        },
+    },
+}
+
+LIST_JOB_STEP_FAILED_RETURN = {
+    "ActionOnFailure": "CONTINUE",
+    "Config": {
+        "Args": ["/usr/lib/spark/bin/run-example", "SparkPi", "10"],
+        "Jar": "command-runner.jar",
+        "Properties": {},
+    },
+    "Id": "s-VK57YR1Z9Z5N",
+    "Name": "calculate_pi",
+    "Status": {
+        "State": "FAILED",
+        "StateChangeReason": {},
+        "FailureDetails": {
+            "LogFile": "s3://fake-log-files/emr-logs/j-8989898989/steps/s-VK57YR1Z9Z5N",
+            "Reason": "Unknown Error.",
+        },
+        "Timeline": {
+            "CreationDateTime": datetime(2016, 6, 20, 19, 0, 18, tzinfo=tzlocal()),
+            "StartDateTime": datetime(2016, 6, 20, 19, 2, 34, tzinfo=tzlocal()),
+        },
+    },
+}
+
+LIST_JOB_STEP_INTERRUPTED_RETURN = {
+    "ActionOnFailure": "CONTINUE",
+    "Config": {
+        "Args": ["/usr/lib/spark/bin/run-example", "SparkPi", "10"],
+        "Jar": "command-runner.jar",
+        "Properties": {},
+    },
+    "Id": "s-VK57YR1Z9Z5N",
+    "Name": "calculate_pi",
+    "Status": {
+        "State": "INTERRUPTED",
+        "StateChangeReason": {},
+        "Timeline": {
+            "CreationDateTime": datetime(2016, 6, 20, 19, 0, 18, tzinfo=tzlocal()),
+            "StartDateTime": datetime(2016, 6, 20, 19, 2, 34, tzinfo=tzlocal()),
+        },
+    },
+}
+
+LIST_JOB_STEP_COMPLETED_RETURN = {
+    "ActionOnFailure": "CONTINUE",
+    "Config": {
+        "Args": ["/usr/lib/spark/bin/run-example", "SparkPi", "10"],
+        "Jar": "command-runner.jar",
+        "Properties": {},
+    },
+    "Id": "s-VK57YR1Z9Z5N",
+    "Name": "calculate_pi",
+    "Status": {
+        "State": "COMPLETED",
+        "StateChangeReason": {},
+        "Timeline": {
+            "CreationDateTime": datetime(2016, 6, 20, 19, 0, 18, tzinfo=tzlocal()),
+            "StartDateTime": datetime(2016, 6, 20, 19, 2, 34, tzinfo=tzlocal()),
+        },
+    },
+}
+
+
+class TestEmrStepsSensor(unittest.TestCase):
```

Comment:
The preference for new-new unit tests is to use `pytest` over `unittest`. Would you mind converting this to `pytest` please?